Misc/NEWS.d/next/Library/2018-01-24-15-20-12.bpo-32391.0f8MY9.rst

```diff
@@ -0,0 +1 @@
+Implement :meth:`asyncio.StreamWriter.wait_closed` and :meth:`asyncio.StreamWriter.is_closing` methods
```

Comment:
One last note: I don't think you want to use ReST formatting in NEWS entries.  This would read better:

```
Implement wait_closed() and is_closing() methods for StreamWriter.
```

Reply:
There is no standard for blurb text, the format of messages is mixed.
Some people use ReST, others prefer plain text.
If you briefly look on these files you'll see both cases.

I have no personal preference.

Reply:
I don't really care.